### ISSUE TITLE
Have you done some experiments about FPN and VOC dataset ?

### ISSUE THREAD

**Issue by soeaver** [endernewton/tf-faster-rcnn#99]
I am very curious the result about FPN and VOC dataset

**Comment by endernewton** [comment]
I don't have FPN implemented, but it should help.
Regarding the baselines setups from FPN, I think VOC does not benefit much from enlarging the image size because it does not have many small objects.

**Comment by soeaver** [comment]
Thanks, I am appreciate for your reply. I have performed some experiments about FPN and VOC on Caffe, the growth of mAP is negligible, just 0.5% on resnet.

**Comment by zacwellmer** [comment]
@soeaver have you posted these results anywhere along with code? I have been looking into implementing an FPN with resnet with this repo.

**Comment by soeaver** [comment]
@zacwellmer  I will release my code and pre-train model after conducting enough experiments.

**Comment by baiyancheng20** [comment]
@soeaver which released code have you used to implement FPN?

**Comment by soeaver** [comment]
@baiyancheng20 py-faster-rcnn

**Comment by SophieZhou** [comment]
@soeaver  Could you share your prototxt file for  FPN with me? I can not train the model successfully. The loss becomes very large using py-faster-rcnn, I think maybe I do not catch up some details of FPN. My email address is jianyingzhou@126.com.
